lib/glific/partners/organization.ex

```diff
@@ -23,13 +23,13 @@ defmodule Glific.Partners.Organization do
   @required_fields [
     :name,
     :shortcode,
-    :email,
     :bsp_id,
     :default_language_id
   ]
 
   # define all the optional fields for organization
   @optional_fields [
+    :email,
```

Comment:
Rn i have made email as optional for organization, because in the new design we only provide email of authority in the end. So current plan is to update that email in the organization table when we get it. Please let me know if we need this change. @AkhileshNegi

Reply:
ok cool